packages/app-next/src/App.tsx

```diff
@@ -16,16 +16,17 @@
 
 import React from 'react';
 import { createApp } from '@backstage/frontend-app-api';
-import { pagesPlugin } from './examples/pagesPlugin';
-import graphiqlPlugin from '@backstage/plugin-graphiql/alpha';
-import techRadarPlugin from '@backstage/plugin-tech-radar/alpha';
-import userSettingsPlugin from '@backstage/plugin-user-settings/alpha';
 import {
   createExtensionOverrides,
   createPageExtension,
 } from '@backstage/frontend-plugin-api';
-import { entityRouteRef } from '@backstage/plugin-catalog-react';
+import graphiqlPlugin from '@backstage/plugin-graphiql/alpha';
 import techdocsPlugin from '@backstage/plugin-techdocs/alpha';
+import techRadarPlugin from '@backstage/plugin-tech-radar/alpha';
+import scmIntegrationsPlugin from '@backstage/integration-react/alpha';
```

Comment:
Because this package is not a role `*-plugin` package, its extensions are not being auto-discovered, that's the reason I'm manually importing it here.